### PR TITLE
Don't print confidential information

### LINKED ISSUE #4482: Username and password of elasticsearch exporter are printed in console on startup
**Describe the bug**

When starting Zeebe the configuration is printed and includes also the configured exporters. The elasticsearch exporter contains username and password to connect to elasticsearch. This information is sensitive and should not be visible in logs. 

**To Reproduce**

Configure a username password for the ES exporter and start the broker.

**Expected behavior**

No sensitive information is visible in the logs.

**Log/Stacktrace**

<details><summary>Full Stacktrace</summary>
 <p>

```
May 07 10:44:54 xxx broker[3103]: "authentication" : {
May 07 10:44:54 xxx broker[3103]: "username" : "elastic",
May 07 10:44:54 xxx broker[3103]: "password" : "changeme"
May 07 10:44:54 xxx broker[3103]: },
May 07 10:44:54 xxx broker[3103]: "index" : {
May 07 10:44:54 xxx broker[3103]: "prefix" : "zeebe-record",
```

</p>
</details>

----

related to [SUPPORT-7686](https://jira.camunda.com/browse/SUPPORT-7686)

### PR DESCRIPTION
**Pull request by ChrisKujawa**
## Description

 The authentication is no longer printed on startup. Do we have an test with elastic auth? So we can verify that this still works?

## Related issues

closes #4482 

## Pull Request Checklist

- [x] All commit messages match our [commit message guidelines](https://github.com/zeebe-io/zeebe/blob/develop/CONTRIBUTING.md#commit-message-guidelines)
- [x] The submitting code follows our [code style](https://github.com/zeebe-io/zeebe/wiki/Code-Style)
- [X] If submitting code, please run `mvn clean install -DskipTests` locally before committing

### CODE REVIEW

File: exporters/elasticsearch-exporter/src/main/java/io/zeebe/exporter/ElasticsearchExporterConfiguration.java [hunk 1]
```diff
@@ -168,16 +174,27 @@ public String toString() {
...
-    public String password;
+    private String username;
+    private String password;
 
     public boolean isPresent() {
       return (username != null && !username.isEmpty()) && (password != null && !password.isEmpty());
     }
 
-    @Override
-    public String toString() {
```
Comment (menski) [inline exporters/elasticsearch-exporter/src/main/java/io/zeebe/exporter/ElasticsearchExporterConfiguration.java:179]: I would propose to still override `toString` and keep it blank or something, with a comment that we should not implement it. That way it's less likely that we later add it accidentally. WDYT?

**Review by menski — APPROVED** [review]
I would propose to override the `toString` as described in my comment, but other then that it's good. Thanks for fixing

**Comment by menski** [comment]
After we merged this change we should backport it to https://github.com/zeebe-io/zeebe/tree/stable/0.23 branch

**Comment by Zelldon** [comment]
Agree thought also about it. Thought also that maybe we could throw an unimplemented exception but maybe it is to rough?

**Comment by menski** [comment]
I think exception is to much. It breaks things like debuggers which call `toString` by default. So let's not go this direction.

**Comment by Zelldon** [comment]
True ok will apply your suggestion later 🙂

**Comment by Zelldon** [comment]
Ok I think my fix is wrong @menski . The problem we have is that the cfg is printed on start up, which means [ExportCfg](https://github.com/zeebe-io/zeebe/blob/develop/broker/src/main/java/io/zeebe/broker/system/configuration/ExporterCfg.java) is actually printed. This class contains a map with the real export config, which is printed as it is. Here we have currently no real possibility to say please dont print confidential information :D

**Comment by Zelldon** [comment]
So it seems to be printed on two places.

**Comment by Zelldon** [comment]
@menski I updated the PR. 

On broker bootstrap the real password and username, if configured, are replaced by three stars.
Looks like this:

```java
  "exporters" : {
    "elasticsearch" : {
      "jarPath" : null,
      "className" : "io.zeebe.exporter.ElasticsearchExporter",
      "args" : {
        "authentication" : {
          "password" : "***",
          "username" : "***"
        },
        "url" : "http://localhost:9200"
      },
      "external" : false
    }
```

We can discuss this, if we want to keep it like this or completely remove them from the output. I had the feeling that it might be better to have it at least this way so the user can see that something he has configured end in the real config.

I also overwrite the `toString` of the `AuthenticationConfiguration`

**Comment by Zelldon** [comment]
@menski do you have time to look at this?

**Review by menski — APPROVED** [review]
bors r+

**Pull request merged**